pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -224,12 +223,12 @@ func (p *prtbLifecycle) ensurePSAPermissions(binding *v3.ProjectRoleTemplateBind
 	// create ClusterRoleBinding to bind the ClusterRole to the user/group
 	psaCRB := &rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: fmt.Sprintf("%s-updatepsa", projectName),
+			Name: fmt.Sprintf("crb-%s-%s-updatepsa", binding.UserName, projectName),
```

Comment:
Can we use `NameForClusterRoleBinding` here as suggested in https://github.com/rancher/rancher/pull/50819#issuecomment-3005666588 ?